.ci/jenkins/scripts/imageUtils.groovy

```diff
@@ -0,0 +1,66 @@
+containerEngine = 'docker'
+containerTlsOptions = ''
+
+void loginRegistry() {
+    loginContainerRegistry(getOperatorImageRegistry(), getOperatorImageRegistryCredentials())
+}
+
+void loginContainerRegistry(String registry, String credsId) {
+    echo "Using credentials ($credsId) to login to registry ($registry)"
+    withCredentials([usernamePassword(credentialsId: credsId, usernameVariable: 'REGISTRY_USER', passwordVariable: 'REGISTRY_PWD')]) {
+        sh "${containerEngine} login ${containerTlsOptions} -u ${REGISTRY_USER} -p ${REGISTRY_PWD} ${registry}"
+    }
+}
+
+String getOperatorImageRegistryCredentials() {
+    return params.OPERATOR_IMAGE_REGISTRY_CREDENTIALS
+}
+
+String getOperatorImageRegistry() {
+    return params.OPERATOR_IMAGE_REGISTRY
+}
+
+String getImageFullName(String namespace, String image, String tag) {
+    return "${namespace}/${image}:$tag"
+}
+
+String getImageFullNameWithRegistry(String registry, String namespace, String image, String tag) {
+    return "${registry}/${getImageFullName(namespace, image, tag)}"
+}
+
+void pullImage(String image) {
+    retry(env.MAX_REGISTRY_RETRIES ?: 1) {
+        sh "${containerEngine} pull ${containerTlsOptions} ${image}"
+    }
+}
+
+void tagImage(String oldImage, String newImage) {
+    sh "${containerEngine} tag ${oldImage} ${newImage}"
+}
+
+void pushImage(String image) {
+    loginRegistry()
+    retry(env.MAX_REGISTRY_RETRIES ?: 1) {
+        sh "${containerEngine} push ${containerTlsOptions} ${image}"
+    }
+}
+
+boolean removeQuayTag(String namespace, String imageName, String tag) {
```

Comment:
I like that one ! 
I guess at some point it could be added to the shared libs: https://github.com/kiegroup/jenkins-pipeline-shared-libraries/blob/main/vars/cloud.groovy :)

Reply:
I am happy to move it there, but for the sake of simplicity, will do that in a separate PR after we merge this one.